docs/spec.md

```diff
@@ -101,14 +102,34 @@ Each JSON message must be an object containing a field called `op` which identif
 - `op`: string `"status"`
 - `level`: 0 (info), 1 (warning), 2 (error)
 - `message`: string
+- `id`: string | undefined. Optional unique identifier that can be used for clearing the status.
```

Comment:
nit: "Optional identifier for the status message. Newer status messages with the same identifier should replace previous messages. `clearStatus` can reference the identifier to indicate a status message is no longer valid."